clang/lib/Basic/SourceManager.cpp

```diff
@@ -996,19 +996,22 @@ CharSourceRange SourceManager::getExpansionRange(SourceLocation Loc) const {
 
 bool SourceManager::isMacroArgExpansion(SourceLocation Loc,
                                         SourceLocation *StartLoc) const {
-  if (!Loc.isMacroID()) return false;
+  if (!Loc.isMacroID())
```

Comment:
All the changes in this file can be reverted because they're just formatting changes, right?